Description: Set the version and the type of the json-api dependency in impl/pom.xml
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
Last-Update: 2021-01-16
--- a/impl/pom.xml
+++ b/impl/pom.xml
@@ -131,6 +131,8 @@
         <dependency>
             <groupId>javax.json</groupId>
             <artifactId>javax.json-api</artifactId>
+            <version>${project.version}</version>
+            <type>bundle</type>
         </dependency>
     </dependencies>
 
